src/app/containers/Bulletin/index.jsx

```diff
@@ -11,11 +10,6 @@ import { tvBulletinItem, radioBulletinItem } from '#models/propTypes/bulletin';
 import { ServiceContext } from '#contexts/ServiceContext';
 
 const BulletinImage = ({ imageValues, lazyLoad }) => {
-  if (!imageValues) {
-    const landscapeRatio = (9 / 16) * 100;
-    return <ImagePlaceholder ratio={landscapeRatio} />;
```

Comment:
Placeholder is removed here since it is likely bulletins don't have images - which would show the BBC Blocks SVG. UX decided that we should remove the placeholder for the Bulletin promo so we don't see the Blocks.